docs(concepts): improve concept pages for elements products with links to relevant docs MTA-1453 (#74)
scaleway/docs-content: compute/apple-silicon/concepts.mdx

## VNC

> Virtual Network Computing (VNC), is a remote desktop-sharing protocol. It allows you to visualize the graphical screen output of a remote computer and to transfer local keyboard and mouse events to the remote computer by using a network connection. The protocol is platform-independent which means that various clients exist for Linux, Windows and macOS based computers. The VNC server used on the M1 is the one directly integrated in the macOS system without any restrictions from our side. Check out our documentation on [how to connect to your Mac Mini M1 via VNC](/compute/apple-silicon/how-to/connect-to-mac-mini-m1/#how-to-connect-via-vnc).

Reviewer: ```suggestion
Virtual Network Computing (VNC), is a remote desktop-sharing protocol. It allows you to visualize the graphical screen output of a remote computer and to transfer local keyboard and mouse events to the remote computer by using a network connection. The protocol is platform-independent which means that various clients exist for Linux, Windows, and macOS-based computers. The VNC server used on the M1 is directly integrated with the macOS system without any restrictions from our side. Check out our documentation on [how to connect to your Mac Mini M1 via VNC](/compute/apple-silicon/how-to/connect-to-mac-mini-m1/#how-to-connect-via-vnc).
```
